monitor/serve-zork-metrics.py

```diff
@@ -0,0 +1,29 @@
+from prometheus_client import start_http_server, Gauge
+import time
+import telnetlib
+
+tn = telnetlib.Telnet("zork", "9000")
+
+getters_gauge = Gauge('number_of_getters', 'Number of uProxy instances connected to this cloud instance.')
```

Comment:
On naming:
 1. Let's devise a prefix so that it's easier to find "our" variables on `/graph`. I suggest `zork_`.
 1. `number_of` is redundant. Suggest `getters`.

In other words, `zork_getters` :-)